book/garbage-collector/README.md

```diff
@@ -651,14 +628,7 @@ Async to capture exceptions and raise them to the appropriate monitor
 for error-handling.  [Async library/finalizers]{.idx}
 
 Let's explore this with a small example that finalizes values of
-different types, some of which are heap-allocated and others which are
-compile-time constants:
-
-<!-- TODO: I'm pretty unsure what this code is getting at.  The
-     "allocated" things aren't necessarily heap allocated: you just
-     can't heap allocate a bool, right?  I tried to figure out how to
-     get my hands on a heap-allocated float, but generally, I just
-     found this whole thing confusing! -->
+different types, all of which are heap-allocated.
```

Comment:
Maybe it's worth keeping some examples of compile-time constants? It's a tricky corner case that people often trip over.

Reply:
deferred to a future edition after some discussion.